gcloudutils/util.go

```diff
@@ -0,0 +1,23 @@
+package gcloudutils
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/api/cloudbuild/v1"
+)
+
+const (
+	nameConst    = "name"
+	versionConst = "version"
+	constraint   = "constraint"
+)
+
+func HandleFailedSourceBuild(ctx context.Context, build *cloudbuild.Build) error {
```

Comment:
No longer used as we don't have any more build which use the cloud `source` for the builds. Could be a good oppurtunity to trim that code